### PR TITLE
fix model code to work with model checkpoints

### PR DESCRIPTION
**Pull request by beasteers**
This PR does 2 things
 - updates the model code to work with the latest checkpoint
 - adds a predict script that lets you run the model on arbitrary audio files

What I could also add:
 - the model weights - we could add these to the repo if you'd like? They're 40mb each
 - some example audio files with plots of the strong annotation outputs - I currently pulled 10 files from freesound (all <10mb) and tried to cover several of the classes but happy to replace with other files too

Let me know if you want me to add either of these

And maybe if you could take a look at model.py, and see if the changes that I made look reasonable. I had to do a bit of guessing because the checkpoint weights had some name differences with the main branch

### CODE REVIEW

**Comment by beasteers** [comment]
These are the example files that I have currently

blender wav
catmeow wav
alarmclock wav
deepalarm wav
dogbark wav
frying wav
razor wav
showerwater wav
speechchild wav
vacuum wav

File: utils/model.py [hunk 1]
```diff
@@ -104,11 +104,11 @@ def __init__(self, in_planes, kernel_size, stride, padding, n_basis_kernels, tem
         if hidden_planes < 4:
             hidden_planes = 4
 
-        if not pool_dim == 'both':
```
Comment (beasteers) [inline utils/model.py:107]: this was a bug I think? I think this would change any value do to `(not pool_dim) == 'both'` => `False == 'both'`

Comment (beasteers) [reply utils/model.py:107]: actually, nvm, just realized that I didn't fix it below and it still works fine (seems weird to me haha)

Comment (frednam93) [reply utils/model.py:107]: yeah it works fine and it's kinda weird haha... I thought the same thing but found out it works find and didn't bother to correct it. Thanks for pointing it out anyway!

**Comment by frednam93** [comment]
Hi! I am glad that you took time on my work :)

Firstly, I am sorry that I gave you the checkpoint that was trained using different model structure. It was trained using not final version of my work, so the conv1d name was fc and bias was True then. Anyway you figured it out and fixed it! Anyway I think I would leave dynamic conv code as it is now, because it's newer version and I think I will not upload the checkpoint that I gave you on this repo.

For gitignore, I am still figuring out what it does. I am still new to github, so give me time to understand it haha. I might update your PR after then.

For prediction script, it's great! but I actually have my own plot script haha. Maybe other people could use yours but I'm not sure if I will add your script on my repo. I'm still not familiar with how things work around in GitHub haha.

Anyway, thanks a lot for your works and suggestions!

**Comment by beasteers** [comment]
> It was trained using not final version of my work

no worries haha, I wasn't sure if it was that or the other way around

> For gitignore, I am still figuring out what it does.

it tells git to ignore certain files which is useful if your project generates certain files that you don't want to upload (e.g. `__pycache__`. It can also be useful if you have files that are either too big or are private

> For prediction script, it's great! but I actually have my own plot script haha.

ok cool I wasn't sure - I just wanted something to generate on a per file basis and didn't see one in the repo already

No worries, thanks for sharing your hard work!

**Pull request closed without merging**